src/main/java/edu/njit/jerse/ashe/Ashe.java

```diff
@@ -100,10 +100,21 @@ public static void run(String root, String targetFile, String targetMethod, Stri
 
         JavaCodeCorrector corrector = new JavaCodeCorrector();
 
-        Path speciminTempDir = corrector.minimizeTargetFile(root, targetFile, targetMethod);
+        Path speciminTempDir;
 
         try {
-            final String sourceFilePath = speciminTempDir.resolve(targetFile).toString();
+            speciminTempDir = corrector.minimizeTargetFile(root, targetFile, targetMethod);
+        } catch (InterruptedException e) {
+            LOGGER.error("Failed to minimize the target file.");
+            LOGGER.info("Exiting...");
```

Comment:
> Wouldn't this produce ASHE to exit #run()?

Yes, but I wouldn't describe that as "ASHE exiting", since I expect `run` to usually be invoked in a loop. You're right that strictly speaking this will cause ASHE to give up on this attempt; maybe we can find a middle ground between "skipping" and "exiting" that conveys that _this run_ of ASHE (on a particular target) has ended early, but that whatever overall process ASHE is running in has not failed? My real concern is that "exiting..." has the connotation of failure, but this is actually a typical, expected "skippable" case - not quite as severe as e.g., a crash within ASHE itself.